Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+node ('virtualbox') {
+
+  def directory = "packer-template"
+  env.PACKER_CACHE_DIR = "${env.JENKINS_HOME}/packer_cache"
+  stage 'Clean up'
+  deleteDir()
+
+  stage 'Checkout'
+  sh "mkdir $directory"
+  dir("$directory") {
+    try {
+        checkout scm
+    } catch (e) {
+        currentBuild.result = 'FAILURE'
+        notifyBuild(currentBuild.result)
+        throw e
+    }
+  }
+  dir("$directory") {
+    stage 'bundle'
+    try {
+        sh "bundle install --path ${env.JENKINS_HOME}/vendor/bundle"
+    } catch (e) {
+        currentBuild.result = 'FAILURE'
+        notifyBuild(currentBuild.result)
+        throw e
+    }
+
+    stage 'rake reallyenglish:build'
+    try {
+      sh 'bundle exec rake reallyenglish:test'
+    } catch (e) {
+        currentBuild.result = 'FAILURE'
+        notifyBuild(currentBuild.result)
+        throw e
+    } finally {
+      sh 'bundle exec rake reallyenglish:clean'
```

Comment:
it seems that the box remains running if a task in the `do_test` fails.
i asked the question to make sure that the boxes are destroyed in case of failures in the `do_test` task.